plugins/versionpress/src/Git/CommitMessage.php

```diff
@@ -57,6 +57,19 @@ public function getSubject()
     }
 
     /**
+     * Returns commit message subject without prefix set in constant VP_COMMIT_MESSAGE_PREFIX
+     * 
+     * @return string
+     */
+    public function getUnprefixedSubject()
```

Comment:
It's not a good time to argue about this (just before the 3.0 release), and if there are refactorings around these things anyway, let's keep the code the way it is (in `CommitMessage`) and reconsider in the future. We need to review the whole VP prefix thing anyway.